### ISSUE TITLE
Single tap for folder pop ups

### ISSUE THREAD

**Issue by geekasso** [8bitPit/Niagara-Issues#1896]
### Is your feature request related to a problem?

I think it would help to have single tap for folder pop ups, similar to the way the app pop up works. 

### Describe the solution you'd like

I would like a defined app to launch when a certain folder pop up is single tapped.

**Comment by topsy-turvy99** [comment]
> ### Is your feature request related to a problem?
> I think it would help to have single tap for folder pop ups, similar to the way the app pop up works.
> 
> ### Describe the solution you'd like
> I would like a defined app to launch when a certain folder pop up is single tapped.

you can acheive the same result by adding additional apps to the app popup.

tapping on it will open the desired app, swiping on it will show the added apps to the pop up.

**Comment by geekasso** [comment]
The problem with app popup, is that you have to close the popup to open that app since it does not get listed once the popup is open, so when jumping between apps the popup has to get closed and reopened.

Perhaps can just add ability to list the app in the app popup so it can be launched without having to be closed.

**Comment by TFackler** [comment]
Thanks for reaching out.

Sadly we won't be adding your requested feature.
We feel like folders should be exactly that, a container for grouping apps together. Having the option to assign an app to open on folder tap seems counter-intuitive. The use case you described is what we intended app-pop ups to be for. 
We also think that not a lot of people have this exact use case.

As a work-around, if you are talking about having these folders on your home screen, would be to have the app you wanted to open on folder tap in your favorites list as well as a folder, which contains said app and all other apps you want to have accessible in the folder. 
This explanation in words feels complicated, so here are screenshots to get the message across (I chose 'Telegram' as the app I want to have quick access to):

![photo_2022-04-02_10-36-23](https://user-images.githubusercontent.com/38538728/161374877-938d359b-6678-4137-b3d7-8b92b73f3676.jpg)
![photo_2022-04-02_10-36-26](https://user-images.githubusercontent.com/38538728/161374882-401ef77d-8f34-415d-80e0-b34aa37a4e25.jpg)
